lib/clientv3/resources.go

```diff
@@ -347,7 +347,11 @@ func (w *watcher) run() {
 
 	for {
 		select {
-		case event := <-w.backend.ResultChan():
+		case event, ok := <-w.backend.ResultChan():
+			if !ok {
+				log.Info("Watcher results channel closed by remote")
```

Comment:
should this be debug?

Reply:
Hmmm, ordinarily I'd say this feels like an info log - we wouldn't generally expect the remote to terminate the watcher suddenly.  That said, we know the CRD-backed resources terminate every minute of inactivity -- so that makes it more borderline for me.

I'll change if you have a strong preference - just let me know @caseydavenport

Reply:
Given this happens a few times every minute, I think it should be debug.